data/core/units/saurians/Spearthrower.cfg

```diff
@@ -0,0 +1,94 @@
+#textdomain wesnoth-units
+[unit_type]
+    id=Saurian Spearthrower
+    name= _ "female^Saurian Spearthrower"
+    race=lizard
+    gender=female
+    image="units/saurians/spearthrower/spearthrower.png"
+    profile="portraits/saurians/skirmisher.webp"
+    hitpoints=39
+    movement_type=lizard
+    movement=7
+    experience=55
+    level=2
+    alignment=chaotic
+    advances_to=Saurian Javelineer
+    cost=23
+    usage=mixed fighter
+    description= _ "Some saurians  with particularly good aim prefer to aid their allies from the backlines rather than to infiltrate enemy formations. Their deadly spear throwing skills allow them to support other saurian warriors from afar."
```

Comment:
Double space after 'saurians', plus 'backlines' should probably 'rear' (I understand English may not be a native language, plus these are supposed to be rewritten anyway).